.github/actions/generic-ci/action.yml

```diff
@@ -162,9 +162,11 @@ runs:
         -DF3D_PLUGIN_BUILD_OCCT=${{ inputs.optional_deps_label == 'optional-deps' && 'ON' || 'OFF' }}
         -DF3D_PLUGIN_BUILD_USD=${{ inputs.optional_deps_label == 'optional-deps' && 'ON' || 'OFF' }}
         -DF3D_PLUGIN_OCCT_COLORING_SUPPORT=${{ runner.os == 'macOS' && 'OFF' || 'ON' }}
+        -DF3D_PLUGIN_BUILD_VDB=${{ inputs.cpu == 'x86_64' && matrix.vtk_version == 'commit' && inputs.optional_deps_label == 'optional-deps' && 'ON' || 'OFF' }}
```

Comment:
sadly I cant make IOOpenVDB work with vtk 9.3.0. Does it matter ?

Reply:
Because of a combination of   https://github.com/AcademySoftwareFoundation/openvdb/issues/1160 and
https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10393

In VTK 9.3.0, we need to be able to find OpenVDB from F3D, and there is only a FindOpenVDB.cmake, no OpenVDB-config.cmake.

Setting CMAKE_MODULE_PATH make exr build fail for some reason, I did not investigate further.

This is fixed in VTK master btw.